llmops/common/prompt_pipeline.py

```diff
@@ -334,6 +344,10 @@ def prepare_and_execute(
         logger.info("Saved the metrics in files in reports folder")
 
 
+# Define a custom argument type for a list of strings
+def list_of_strings(arg):
+    return arg.split(',')
```

Comment:
this is duplicated in various python files. opportunity to create a string utils class for this to be reused?

Reply:
Fixed